tests/sentry/issues/auto_source_code_config/test_process_event.py

```diff
@@ -106,23 +129,23 @@ def _process_and_assert_configuration_changes(
             starting_repositories_count = Repository.objects.all().count()
             starting_code_mappings_count = RepositoryProjectPathConfig.objects.all().count()
             event = self.create_event(frames, platform)
-            dry_run_code_mappings, in_app_stack_trace_rules = process_event(
```

Comment:
This method can return both code_mappings that would have been created (dry-run) and created ones.